app/telegram/handlers.py

```diff
@@ -0,0 +1,55 @@
+from .models import Chat
+from .pylander import pylander
+
+
+class MessageHandler:
+    COMMANDS = ['/start', '/logout', '/show_events', '/new_event']
+
+    def __init__(self, chat: Chat):
+        self.chat = chat
+        self.handlers = {}
+        for method in dir(self):
```

Comment:
This is a cool way, but it might be used to inject code. Prefer building this by specifying hard-coded names